### ISSUE TITLE
The question about `retrainLayerIndex = -4` in code

### ISSUE THREAD

**Issue by tianyunzhe** [shrezaei/Target-Agnostic-Attack#1]
Hi. First of all, your paper has been received by ICLR2020, congratulations! The Security of Deep Learning Methodologies you mentioned is a meaningful research topic. What’s more, the design of brute-force attack you proposed is efficient. But I have one question in your code.
In your code about face recognition, when you launch the attack, the last layer of feature extractor shouldn’t be FC6/relu, that is, activation vector shouldn’t be the output of FC6/relu. The reason why you set `retrainLayerIndex = -4` in your code, maybe you have known when you re-trained the model, you remove the last FC layer and the softmax layer from the pre-trained VGG face model to make a feature extractor. But in your assumptions, we know nothing about the new model ,that is,  we don’t know the feature extractor of re-trained model and we don't know the last layer of feature extractor. So, in my opinion, when you launch the brute-force attach, the last layer of feature extractor should be the penultimate layer (FC7/relu) rather than FC6/relu and the activation vector should be the scores at the penultimate layer. And in the introduction in your paper, you also  mentioned 

> We call the scores at the penultimate layer activation vector and the part of the model that produces the activation vector feature extractor. 

Looking forward to your reply! Thank you !
